cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
do we need this json?

Reply:
I dont see this

Reply:
Yes, cypress is now configured to look for the tests in the `tests` folder